### ISSUE TITLE
Minimize to taskbar

### ISSUE THREAD

**Issue by Grekkq** [jenius-apps/ambie#4]
Great app but i think there should be an option (simple checkbox will do the job) to minimize app to taskbar, it's annoying to have another app that works in a background as a open window, especially when new windows update add every page in edge to alt+tab menu.

**Comment by dpaulino** [comment]
When you say minimize to the task bar, do you mean minimize to this section?
![image](https://user-images.githubusercontent.com/7649589/97926428-f82bb780-1d17-11eb-86f2-f28126568055.png)

**Comment by VinhDevNguyen** [comment]
@dpaulino I think it will be better if you add minimize feature like you said

**Comment by Grekkq** [comment]
@dpaulino yes, similar to Spotify
![image](https://user-images.githubusercontent.com/32731729/97969397-485e5480-1dc0-11eb-9ab7-929f0871ee35.png)
![image](https://user-images.githubusercontent.com/32731729/97969686-b30f9000-1dc0-11eb-96d8-ddcec9be04cf.png)

**Comment by MarcAnt01** [comment]
I think this is not a good solution, also I think using Desktop Bridge will break the possibility of having the sounds playing with the screen off

**Comment by dpaulino** [comment]
@Grekkq @VinhDevNguyen thanks for the comments. System tray is not easy to work with. It will degrade other UX aspects of Ambie (such as what MarcAnt01 said), and it will break xbox support (or will make it difficult to). For these reasons, please know that minimizing to taskbar will be a low priority for the next little while. That said, I'd like to keep this issue open to track this request. Sorry if this is a downer, but I wanted to be honest and I wanted to set your expectations.

**Comment by pedrozaalex** [comment]
@dpaulino Hey, I'm really interested in this feature and would love to help out with it if possible/needed.

I think it's super worth adding and it fits in so well with Ambie's motto of _less is more_: if the app is out of your desktop and out of your sight, it's also out of your mind, reducing cognitive load.

If you still want to go forward with it, what would be the priorities here? Maintaining multiplatform support?

P.S.: sorry, if I'm going about this the wrong way, I'm new to contributing to open source, and would really enjoy this feature.
P.P.S.: thanks and congrats for making Ambie, I love it to bits!

**Comment by dpaulino** [comment]
@pedrozaalex Thanks for the kind words! 

I think a proof of concept would be interesting to see if you can provide one. However, I'm skeptical that the cost would be worth it, particularly since only a handful of people asked me for this feature out of 14000+ users. Also, I believe wapproj will be required for this, and wapproj is a pain to work with and it has pitfalls such as compilation issues, arm64 issues, and winui issues (all issues I faced first-hand). 

Feel free to fork ambie and build a proof of concept. I'd be happy to re-evaluate once your PoC is complete. But I can't make promises on if I would agree to a merge. It would have to depend on the maintainability costs and any UX regressions. 

As said in a previous comment, sorry if I sound discouraging about all this. I just wanted to be upfront about my reluctance with using and maintaining wapproj. Ideally, Windows should just build a modern version of systray that works directly with WinRT so wapproj isn't needed. But anyway, it would be cool to see a PoC.

**Comment by dpaulino** [comment]
Also, I should add that even if the PoC works well, there will be numerous design questions that will need to be addressed before further implementation should be done. "Just before we can, doesn't mean we should." The UX has to be well thought out before something like this were fully implemented.

**Comment by nhan000** [comment]
Just want to leave a request for this enhancement. Would really make a better experience.

**Comment by giseledute** [comment]
It would be incredible if you could see the time count/focus on the tray.

**Comment by ultrapassado** [comment]
For me this feature is essential. If I'm not able to close/minimize the app to tray, I much prefer to play the sounds on YouTube, because then it's only a tab in the browser, not a full window opened.
